src/cli/common.rs

```diff
@@ -424,14 +424,20 @@ pub fn list_installed_components(toolchain: &Toolchain<'_>) -> Result<()> {
     Ok(())
 }
 
-pub fn list_toolchains(cfg: &Cfg, is_verbose: bool) -> Result<()> {
-    let toolchains = cfg.list_toolchains()?;
-    let toolchain_info = if is_verbose {
-        format!("\t{}", &cfg.rustup_dir.display())
+fn print_toolchain_path(cfg: &Cfg, toolchain: String, if_default: &str, verbose: bool) {
```

Comment:
Also, I think I'd prefer if this function took `toolchain` as `&str` even though in theory it's the only consumer of the `String` produced in the calling function -- just to futureproof things a little.